x-pack/test/alerting_api_integration/spaces_only/tests/alerting/snooze.ts

```diff
@@ -128,8 +133,111 @@ export default function createSnoozeRuleTests({ getService }: FtrProviderContext
         supertest,
         spaceId: Spaces.space1.id,
         type: 'alert',
-        id: createdAlert.id,
+        id: createdRule.id,
       });
     });
+
+    it('should not trigger actions when snoozed', async () => {
+      const { body: createdAction, status: connStatus } = await supertest
+        .post(`${getUrlPrefix(Spaces.space1.id)}/api/actions/connector`)
+        .set('kbn-xsrf', 'foo')
+        .send({
+          name: 'MY action',
+          connector_type_id: 'test.noop',
+          config: {},
+          secrets: {},
+        });
+      expect(connStatus).to.be(200);
+      objectRemover.add(Spaces.space1.id, createdAction.id, 'action', 'actions');
+
+      log.info('creating rule');
```

Comment:
**question:** should these logs be `info` or `debug`? I believe CI used to or still captures the debug logs whenever a flaky or failed test happens.